src/pages/governance/_components/VotingResult.tsx

```diff
@@ -192,65 +183,34 @@ export function VotingResult({
                   >
                     Details
                   </div>
-                  <div
-                    className={classNames(
-                      "lg:hidden md:block hidden pb-3 border-b-[0.5px] dark:border-dark-gray-300 mb-3"
-                    )}
-                  >
-                    <div className="flex flex-col gap-y-1">
-                      <LabelWithInfoTooltipAndChecks
-                        labelTitle="Approval rate"
-                        value={percYes}
-                        comparatorValue={
-                          new BigNumber(
-                            proposal.approvalThreshold.replace("%", "")
-                          )
-                        }
-                        suffix="%"
-                        toolTipDesc="The percentage of required “yes” votes for proposal to be considered accepted."
-                        decimalPlace={2}
-                      />
-
-                      <NumericFormat
-                        value={proposal.approvalThreshold}
-                        fixedDecimalScale
-                        thousandSeparator=","
-                        displayType="text"
-                        prefix="(Min. "
-                        suffix="%)"
-                        className="text-xs text-gray-500 dark:text-dark-gray-500 w-full text-right"
-                      />
-                    </div>
-                  </div>
                 </>
               )}
 
-              <div className="flex flex-col">
+              <div className="flex flex-col gap-y-5">
                 <LabelWithInfoTooltipAndChecks
-                  labelTitle="Neutral votes"
-                  value={BigNumber(voteCounts.neutral)}
-                  // toolTipDesc="Included as part of the total votes submitted, excluded from min. approval." // TODO: uncomment when blockchain fixes neutral votes as no bug
+                  labelTitle="Min. required approval rate"
+                  value={
+                    new BigNumber(proposal.approvalThreshold.replace("%", ""))
```

Comment:
i see we have `new BigNumber(..` and sometimes `BigNumber(..` in this code, let's be consistent

Reply:
removed all BigNumber to use new BigNumber